doc/DATA_MODEL.md

```diff
@@ -28,6 +28,26 @@ Nodes may also be redirects, though this use is legacy only. More documentation
 
 Notes (type `note`) can also be a sub-type called Questions, if they have a tag starting with `question:` -- which gives them extra features such as the ability to have Answers (see below).
 
+Node `status` -- a property of `Node`, can be:
+
+* 0: banned
+* 1: normal
+* 3: draft
+* 4: moderated
```

Comment:
can we add `moderated -- i.e. by a first-time poster, and has not yet been "approved"`

Reply:
ok, i will change it. Thanks!